### PR TITLE
Fix TES compliance tests

### PR DESCRIPTION
**Pull request by yuliadub**
ever since the tes compliance test repo was renamed, the tes tests have not been working due to this change. This PR will fix that. 

issues related to this: 
https://github.com/microsoft/ga4gh-tes/issues/457
https://github.com/microsoft/ga4gh-tes/issues/397
https://github.com/microsoft/ga4gh-tes/issues/393

### CODE REVIEW

File: src/TesApi.Web/tes-compliance-test/entrypoint.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ teshostname=$(jq -r '.TesHostname' TesCredentials.json)
...
 
-# This file will be used to run tes-compliance-tests as part of the ADO release
-# TES-compliance-suite repo provides a default entry point script, but since TES uses basic auth it needs to be replaced to add approprite credentials
-# https://github.com/elixir-cloud-aai/tes-compliance-suite
-tes-compliance-suite report --server http://$tesuser:$tespassword@$teshostname/ --tag all --version 1.0.0 --output_path results
\ No newline at end of file
+# This file will be used to run tes compliance tests as part of the ADO release
+# openapi-test-runner repo provides a default entry point script, but since TES uses basic auth it needs to be replaced to add approprite credentials
+# https://github.com/elixir-cloud-aai/openapi-test-runner
+openapi-test-runner report --server http://$tesuser:$tespassword@$teshostname/ --tag all --version 1.0.0 --output_path results
```
Comment (yuliadub) [inline src/TesApi.Web/tes-compliance-test/entrypoint.sh:9]: i asume we are still on version tes 1.0.0, is that correct?

**Comment by yuliadub** [comment]
after these changes tests ran successfully: 

image

File: src/TesApi.Web/tes-compliance-test/entrypoint.sh (hunk 1 above)
Comment (MattMcL4475) [reply src/TesApi.Web/tes-compliance-test/entrypoint.sh]: Great point, yes, and I wonder if we could loop through each TES version (1.0.0, 1.1.0), run the `openapi-test-runner`, and update the status of a badge on the readme that says "✔️ GA4GH TES 1.0.0", "✔️ GA4GH TES 1.1.0"

**Review by MattMcL4475 — APPROVED** [review]

File: src/TesApi.Web/tes-compliance-test/entrypoint.sh (hunk 1 above)
Comment (yuliadub) [reply src/TesApi.Web/tes-compliance-test/entrypoint.sh]: we certainly can - but this is a bit more of a lift. Great opportunity for someone to pick this up and learn how these tests work (so I am not the only one with this knowledge). I'll make an issue to track this and happy to help if any road blocks are hit.

**Pull request merged**